lib/game_collection.rb

```diff
@@ -94,4 +93,49 @@ def biggest_blowout
     end.difference_between_score
     games_difference
   end
+
+  
+
+  def find_away_defense_goals(away_team_id)
```

Comment:
Could possibly be confusing method names. Logic is sound!

Reply:
I agree.